android/sdk/src/main/java/org/jitsi/meet/sdk/JitsiMeetOngoingConferenceService.java

```diff
@@ -95,26 +98,45 @@ private static void doLaunch(Context context, HashMap<String, Object> extraData)
 
 
     public static void launch(Context context, HashMap<String, Object> extraData) {
-        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.TIRAMISU) {
-            PermissionListener listener = new PermissionListener() {
-                @Override
-                public boolean onRequestPermissionsResult(int i, String[] strings, int[] results) {
-                    if (results.length > 0 && results[0] == PackageManager.PERMISSION_GRANTED) {
+        List<String> permissionsList = new ArrayList<>();
+
+        PermissionListener listener = new PermissionListener() {
+            @Override
+            public boolean onRequestPermissionsResult(int i, String[] strings, int[] results) {
+
+                if (results.length > 0) {
+
+                    if (results[0] == PackageManager.PERMISSION_GRANTED
+                        && results[1] == PackageManager.PERMISSION_GRANTED) {
                         doLaunch(context, extraData);
+                        JitsiMeetLogger.w(TAG + " Service launched, permissions were granted");
+                    } else {
+                        JitsiMeetLogger.w(TAG + " Couldn't launch service, permissions were not granted");
                     }
-
-                    return true;
                 }
-            };
 
+                return true;
+            }
+        };
+
+        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.TIRAMISU) {
```

Comment:
Isn't the audio permission only required for 34 but not before, and notifications were neded since 30 or something? That suggests we have 3 cases:

- no need for permissions at all
- need for media proyection
- need for media proyection AND microphone

Reply:
Notification were needed since 33. Don't you think it would be much simpler to include both for >= 33?

Reply:
We can do that yeah.

Reply:
Yeah.